src/SALib/sample/sobol.py

```diff
@@ -31,35 +31,38 @@ def sample(
     rows, where ``D`` is the number of parameters.
     If `calc_second_order` is `True`, the resulting matrix has ``N * (2D + 2)``
     rows.
+
     These model inputs are intended to be used with
     :func:`SALib.analyze.sobol.analyze`.
 
+
     Notes
     -----
     The initial points of the Sobol' sequence has some repetition (see Table 2
-    in Campolongo [1]_), which can be avoided by scrambling the sequence.
+    in Campolongo [1]__), which can be avoided by scrambling the sequence.
```

Comment:
This should have been ok with the way references where listed in the reference section.

The fix in general works, an alternative was to make use of the refs in the text. Still I am surprised it was actually an issue in the first place as we have that in SciPy and don't call all the refs in the text.

Reply:
I wonder what plugins are in use for SciPy for the number style referencing?

At the moment, we get a large number of errors related to duplicate or unused references because the plugin expects the references to be globally unique. Most of the suggested solutions seem to be to switch to using bibtex style referencing, or to remove the reference system/plugin and just use simple numbered lists.

Reply:
@ConnectedSystems sorry for the delay. Right this is coming from numpydoc, see: https://numpydoc.readthedocs.io/en/latest/format.html#references

Maybe something to consider?

Reply:
Ah, so any references has to be used under the  `Notes` section? That can be done I guess...

Reply:
It's not an obligation. You can have refs which are not being called (we have plenty in SciPy). Although might make sense long term to try to use what we cite, like in an paper.

Also you can call refs in the whole docstring, not only the Notes section.